core/tests/behavior/write.rs

```diff
@@ -47,6 +47,7 @@ pub fn behavior_write_tests(op: &Operator) -> Vec<Trial> {
         test_create_dir,
         test_create_dir_existing,
         test_write_only,
+        test_write_empty_file,
```

Comment:
No all services support write empty file, we should add a new capability for this. Maybe it's better to split into a new PR instead?

Reply:
I've removed it. Will do this in a new PR.